ckanext/showcase/commands/migrate.py

```diff
@@ -41,18 +49,17 @@ def command(self):
             self.migrate()
         elif cmd == 'make_related':
             self.make_related()
+        elif cmd == 'delete':
+            self.delete()
```

Comment:
That's embarassing.. Relicts of debugging in a time long long ago..  Never to be seen again 😛